shared/src/main/scala/NumberHelpers.scala

```diff
@@ -147,7 +147,13 @@ object NumberHelpers:
     if value == VNum(0) then VList(List(VNum(0))*)
     else if base.toBigInt == -1 then
       VList.from(
-        Seq.fill(value.toInt.abs)(Seq(1, 0)).flatten.map(VNum(_))
+        Seq
+          .fill(value.toInt.abs)(Seq(1, 0))
+          .flatten
+          .map(VNum(_))
+          .dropRight(
+            if value.toBigInt > 0 then int2Integer(1) else int2Integer(0)
```

Comment:
`int2Integer` should not be necessary. That converts `Int`s to `java.lang.Integer`s, and Java `Integer`s should not be appearing anywhere in our code. Does removing those calls not work?
```suggestion
            if value.toBigInt > 0 then 1 else 0
```

Reply:
As you probably discovered the linter was crying about it 😛